test/libsolidity/semanticTests/array/invalid_encoding_for_storage_byte_array.sol

```diff
@@ -0,0 +1,92 @@
+contract C {
+    bytes public x = "abc";
+    bytes public y;
+    function invalidateXShort() public {
+        assembly { sstore(x.slot, 64) }
+        delete y;
+    }
+    function invalidateXLong() public {
+        assembly { sstore(x.slot, 5) }
+        delete y;
+    }
+    function abiEncode() public view returns (bytes memory) { return x; }
+    function abiEncodePacked() public view returns (bytes memory) { return abi.encodePacked(x); }
+    function copyToMemory() public view returns (bytes memory m) { m = x; }
+    function indexAccess() public view returns (byte) { return x[0]; }
+    function assignTo() public { x = "def"; }
+    function assignToLong() public { x = "1234567890123456789012345678901234567"; }
+    function copyToStorage() public { y = x; }
+    function copyFromStorageShort() public { y = "abc"; x = y; }
+    function copyFromStorageLong() public { y = "abc"; x = y; }
```

Comment:
These two have identical code. Was it meant to be `y = "1234567890123456789012345678901234567"` in the long case?